core/src/main/scala/io/projectglow/vcf/VCFFileFormat.scala

```diff
@@ -273,6 +273,23 @@ object VCFFileFormat {
       false
     }
   }
+
+  def requireWritableAsVCF(schema: StructType): Unit = {
+    val baseRequiredFields =
+      Seq(VariantSchemas.contigNameField, VariantSchemas.startField, VariantSchemas.refAlleleField)
+    val requiredFields = if (schema.exists(_.name == VariantSchemas.genotypesFieldName)) {
+      baseRequiredFields :+ VariantSchemas.alternateAllelesField
```

Comment:
Is this strictly true? It looks like we can successfully make `Genotype`s as long as the appropriate allele exists in the `VariantContext` (i.e. only called reference allele).

Reply:
As discussed today, I think it's better to disallow this. It's possible to form a variant context, but unlikely to result in the behavior users expect. They can always add a dummy `alternateAlleles` if it's really what they want.